pandas/tests/frame/test_rank.py

```diff
@@ -266,3 +266,56 @@ def _check2d(df, expected, method='average', axis=0):
                 continue
             frame = df if dtype is None else df.astype(dtype)
             _check2d(frame, results[method], method=method, axis=axis)
+
+
+# GH15630, pct should be on 100% basis when method='dense'
+@pytest.mark.parametrize('frame, exp', [
```

Comment:
can you make frame a fixture, its seems the same for each of these test functions. just the results are diff.

Reply:
or maybe better is to inline it inside the function and use a single parameterize on the ``method, exp``

Reply:
I like your second idea better.